### DISCUSSION TITLE
Receiver not working after updating firmware

### DISCUSSION THREAD

**Discussion by AirbagUK** [darthcloud/BlueRetro#909] [Q&A]
Please help!

I've recently purchased a Blueretro mini PlayStation reciver. I downloaded NodeMCU-PyFlasher along with the latest firmware v1.9.1_hw1. I followed the instructions that came with the device and updated using the file BlueRetro_hw1_playstation.bin

Since doing this my device has stopped working, the red and blue led are both on at the same time and I cannot pair a controller or pair the device via bluetooth to my PC.

What's gone wrong, and how can I fix it?

Many thanks
Andrew

**Comment by darthcloud** [comment]
Use the real instruction:
https://github.com/darthcloud/BlueRetro/wiki/Flashing-firmware-Windows-10

**Reply by AirbagUK** [reply]
Legend! 

That's fixed it, thank you...

**Reply by NioDio10** [reply]
I had the same issue and this helped me out, thank you.

**Reply by trabi1986** [reply]
Thank you very much!
